src/tests/tracing/eventpipe/common/ExposedSocketNetworkStream.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Sockets;
+
+namespace Tracing.Tests.Common
+{
+    internal sealed class ExposedSocketNetworkStream :
```

Comment:
Is this new class really necessary? Is it bad to just use NetworkStream in IpcTransport?